### PR TITLE
Add exceptions messages to Resource file

### PR DESCRIPTION
**Pull request by hishamco**

### CODE REVIEW

File: src/Microsoft.AspNet.Session/DistributedSession.cs [hunk 1]
```diff
@@ -62,13 +62,13 @@ public void Set(string key, [NotNull] byte[] value)
             if (encodedKey.KeyBytes.Length > KeyLengthLimit)
             {
                 throw new ArgumentOutOfRangeException(nameof(key),
-                    string.Format("The key cannot be longer than '{0}' when encoded with UTF-8.", KeyLengthLimit));
+                    string.Format(Resources.Exception_KeyLengthIsExceeded, KeyLengthLimit));
```
Comment (Tratcher) [inline src/Microsoft.AspNet.Session/DistributedSession.cs]: There's an overload that does not require string.Format. See Resources.FormatException_KeyLengthIsExceeded

Comment (hishamco) [inline src/Microsoft.AspNet.Session/DistributedSession.cs]: You are right
thanks @Tratcher

**Comment by hishamco** [comment]
the PR has been updated

**Comment by Eilon** [comment]
:shipit:

**Comment by Tratcher** [comment]
Rebased, squashed, and merged.

**Comment by hishamco** [comment]
Thanks @Tratcher

**Pull request closed without merging**